recipes/hhsuite/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "v3.1.0" %}
+{% set sha256 = "e67527c8dd175f5730ca877aa506bc7aa4178cb0154d5b1a2794e1a484956c83" %}
+
+package:
+  name: hhsuite
+  version: {{ version }}
+
+build:
+  number: 1
+
+source:
+  url: https://github.com/soedinglab/hh-suite/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make
+  host:
+    - openmp
+    - perl
```

Comment:
Why is perl and python needed here? Is this also a run time dependency?

Reply:
We have scripts in hhsuite that are written in python and perl. https://github.com/soedinglab/hh-suite/tree/master/scripts

Reply:
ok, then this should be added as run-time deps as well.